File: Upcoder/Khối 11/Chương 1 Làm Quen/Gạch sọc/K11_GachSoc.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    double a;
    cin >> a;
    double Shcn = a * a;
    double Shtron = M_PI * (a / 2) * (a / 2);
    cout << Shcn - Shtron;
    return 0;
}

File: Upcoder/Khối 11/Chương 1 Làm Quen/Gạch sọc/K11_GachSoc.cs
using System;
namespace Gachsoc
{
    public class Program
    {
        static void Main(string[]args)
        {
            double x = double.Parse(Console.ReadLine());
            double Shcn = x * x;
            double Shtron = Math.PI * (x / 2) * (x / 2);
            Console.Write(Shcn - Shtron);
        }
    }
}